Steps to Deploy Locally (#11)

gaurang7goel opened the pull request:
## My first image
![bicycle-1280x720_v4rvgkv](https://user-images.githubusercontent.com/25103902/33525748-22d7766c-d85c-11e7-8c01-22d5a9cbe953.jpg)

_Its compression_
![jpeg image 320 x 180 pixels](https://user-images.githubusercontent.com/25103902/33525750-236fee56-d85c-11e7-9f03-2c9c81f485fa.jpg)

## My second image
![thumb-1920-411820](https://user-images.githubusercontent.com/25103902/33525753-268684ba-d85c-11e7-8c1e-fcb2c96c08e4.jpg)
_Its compression_
![jpeg image 320 x 200 pixels](https://user-images.githubusercontent.com/25103902/33525751-250e284a-d85c-11e7-988e-c758874b2255.jpg)

jatinluthra14 committed:
Steps to Deploy Locally (#11)

gaurang7goel commented:
my compressor-head project.

Commit:
Add support Travis CI adn requirements.txt

Commit:
Add travis status icon to README

ssudhanshu488 commented:
Image before compression and link of this image is : https://media.wired.com/photos/593222b926780e6c04d2a195/master/w_2400,c_limit/Zuck-TA-AP_17145748750763.jpg
![zuck-ta-ap_17145748750763](https://user-images.githubusercontent.com/34166698/33528998-e3f219aa-d88f-11e7-8de3-43e929ffb07a.jpg)

Image after compression and link of this is : http://compressor-head.appspot.com/image/?image_url=https://media.wired.com/photos/593222b926780e6c04d2a195/master/w_2400,c_limit/Zuck-TA-AP_17145748750763.jpg&width=500&height=350&format=jpg
![compressor-head appspot com](https://user-images.githubusercontent.com/34166698/33529014-365a172e-d890-11e7-8b7c-7afcc98e2ee0.jpeg)


Second Image before compression and link of this image is : http://pmindia.gov.in/wp-content/uploads/2014/06/High1.jpg
![high1](https://user-images.githubusercontent.com/34166698/33529036-b9d27024-d890-11e7-8022-42119c866e4b.jpg)

Second Image after compression and link of this is : http://compressor-head.appspot.com/image/?image_url=http://pmindia.gov.in/wp-content/uploads/2014/06/High1.jpg&width=500&height=240&format=jpg
![compressor-head appspot com1](https://user-images.githubusercontent.com/34166698/33529067-4e0a16fc-d891-11e7-900c-e7b18485b8dd.jpeg)

Commit:
Update readme.md

Commit:
Update readme.md

Commit:
Merge pull request #29 from Hatollint/addtravis

Add support Travis CI and requirements.txt

HITGIF committed:
Add Deployment Guide for MacOS

Commit:
Integrate Codacy to this project (#39)

* Add Badge Codacy

* Create .codacy.yml

opticod committed:
Merge branch 'master' into patch-1

opticod committed:
Merge branch 'master' into patch-1

opticod committed:
Merge pull request #44 from HITGIF/patch-1

GCI: Update documentation

Bolt108 committed:
Updated README.md to include support for deployment of local server on Mac. (#31)

RichardNgai committed:
Removed one blank line (#57)

elskhn committed:
refactor: README typos and grammar fixes (#56)

fix README typos and grammar errors

Commit:
Add guide (#54)

Vishwas-Adiga committed:
Update compressor-head documentation (#62)

MarvinRucinski committed:
Update readme.md (#64)

zwliew committed:
GCI: Update documentation for compressor-head (#69)

* Convert license format to one recognizable by GitHub

* Improve documentation

 * Improve the organization of info
 * Present the most important info first - the usage guide.
   Lead the reader to the setup and contribution guides later on
   if they are interested
 * Clean up some formatting
 * Improve copyright presentation
 * Remove Travis and Codacy build status badges (they aren't set up)

zwliew committed:
Allow both width and height of 0 (#68)

Setting both width and height to 0 simply does not resize
the image at all.

SachinRammoorthy committed:
Updated Documentation (#71)

* Updated readme.md

* Updated CONTRIBUTING.md

* Corrected typo

* Updated CONTRIBUTING.md

* Updated SETUP.md

* Updated CONTRIBUTING.md

sidhyatikku committed:
Documentation update for Comp Head (#76)

* Update readme.md

* Update CONTRIBUTING.md

* Update SETUP.md

RaghavAwasthi committed:
Updated Documentation

Commit:
Merge pull request #79 from RaghavAwasthi/master

Updated Documentation

zwliew committed:
Correct setup instructions for Windows (#70)

In addition, update the Google App Engine SDK references to
version 1.9.64.

SachinRammoorthy committed:
docs: Updated macOS documentation (#81)

vicens-aniol committed:
Write/update documentation of compressor-head project deployment.

Set up *Compressor-head* in Google Cloud Platform documentation.

Commit:
Merge pull request #84 from vicens-aniol/docemployment

Write/update documentation of compressor-head project deployment. - GCI Task
